### PR TITLE
Enforce user selection for welcome wizard privacy and data sharing - MAILPOET-4681

### PR DESCRIPTION
**Pull request by triple0t**
## Description

Enforce user selection for welcome wizard privacy and data sharing page

We are making the two buttons mandatory. Users have to select either yes or no before they can proceed with the welcome wizard.

## Code review notes

Kind reminder to run `./do compile:js`

## QA notes

Check ticket description

## Linked tickets

[MAILPOET-4681](https://mailpoet.atlassian.net/browse/MAILPOET-4681)

### CODE REVIEW

File: mailpoet/assets/js/src/wizard/steps/usage_tracking_step.jsx [hunk 1]
```diff
@@ -6,13 +6,25 @@ import { Button } from 'common/button/button';
 import { Heading } from 'common/typography/heading/heading';
 import { YesNo } from 'common/form/yesno/yesno';
 
+const checkForNullOrUndefined = (value) =>
```
Comment (samnajian) [inline mailpoet/assets/js/src/wizard/steps/usage_tracking_step.jsx]:
```suggestion
const isNullOrUndefined = (value) =>
```

**Review by samnajian — CHANGES_REQUESTED** [review]

**Review by samnajian — APPROVED** [review]
LGTM!

**Review by Aschepikov — APPROVED** [review]
Verified, GJ!

**Review by Aschepikov — APPROVED** [review]
Verified, GJ!

**Pull request merged**